services/gradle-plugin-portal/gradle-plugin-portal.service.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const { redirector } = require('..')
+
+module.exports = redirector({
+  category: 'version',
+  isDeprecated: false,
+  route: {
+    base: 'gradle-plugin-portal/v',
+    // TODO: add /:versionPrefix? when maven-metadata have versionPrefix attribute.
+    pattern: ':pluginId',
+  },
+  examples: [
+    {
+      title: 'Gradle Plugin Portal',
+      namedParams: {
+        pluginId: 'com.gradle.plugin-publish',
+      },
+      staticPreview: {
+        label: 'plugin portal',
+        message: 'v0.14.0',
+        color: 'blue',
+      },
+    },
+    /*
+    {
+      title: 'Gradle Plugin Portal with version prefix filter',
```

Comment:
Please remove this commented example for now. When and if we get round to adding a version prefix attribute to maven-metadata, it's enough enough to re-add at that point, without the risk of the commented code having gone stale in the meantime. 😉

Reply:
I removed this comment.